Content.Shared/Whitelist/EntityWhitelistSystem.cs

```diff
@@ -51,8 +51,11 @@ public bool IsValid(EntityWhitelist list, EntityUid uid)
         {
             var regs = StringsToRegs(list.Components);
 
-            list.Registrations ??= new List<ComponentRegistration>();
-            list.Registrations.AddRange(regs);
+            if (list.Registrations == null)
+            {
+                list.Registrations = new List<ComponentRegistration>();
+                list.Registrations.AddRange(regs);
+            }
         }
```

Comment:
This is the same code?

Reply:
Line 57 here keeps piling up the same `regs` onto existing List with every function call.

Reply:
better yet put the `var regs` line in the block as well